src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_validators.py

```diff
@@ -429,6 +442,64 @@ def _validate_storage_pool_size(storage_pool_size, storage_pool_type):
             )
 
 
+def validate_enable_azure_container_storage_v2_params(
+    is_v2_extension_installed,
+    is_v1_extension_installed,
+    v1_extension_version,
+    storage_pool_name,
+    storage_pool_sku,
+    storage_pool_option,
+    storage_pool_size
+):
+    if is_v1_extension_installed:
+        raise InvalidArgumentValueError(
+            f'Azure Container Storage version {v1_extension_version} is already installed on the cluster '
+            'which is why Azure Container Storage v2 cannot be enabled. '
```

Comment:
np: Similarly, we could update this error to match with the v2: 

```Failed to enable Azure Container Storage V2 because {v1_extension_version} version of Azure Container Storage V1 is already enabled on the cluster. To enable Azure Container Storage V2, please first disable V1 using the --disable-azure-container-storage flag. 
Note: Disabling V1 may affect existing workloads that depend on Azure Container Storage V1. ```